### PR TITLE
Spelling fixes, latest lints, move to minilibs (drop parts)

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: example/template.dart [hunk 1]
```diff
@@ -21,7 +21,7 @@ class TemplateBenchmark extends BenchmarkBase {
   @override
   void setup() {}
 
-  // Not measures teardown code executed after the benchark runs.
+  // Not measures teardown code executed after the benchmark runs.
```
Comment (devoncarew) [inline example/template.dart:24]: This sentence doesn't read well even w/ the spelling fix :) Should this be 'Note that the time spent in teardown doesn't contribute to the benchmark time.' ?

File: lib/src/async_benchmark_base.dart [hunk 2]
```diff
@@ -30,10 +30,10 @@ class AsyncBenchmarkBase {
   /// Not measured setup code executed prior to the benchmark runs.
   Future<void> setup() async {}
 
-  /// Not measures teardown code executed after the benchark runs.
+  /// Not measures teardown code executed after the benchmark runs.
```
Comment (devoncarew) [inline lib/src/async_benchmark_base.dart:33]: Same comment as above.

**Review by devoncarew — APPROVED** [review]
lgtm, but note the two grammar fixes

**Comment by kevmoo** [comment]
Gah! auto-merge strikes again!

**Comment by devoncarew** [comment]
Opps - I shouldn't have hit the 'approved' button -

**Comment by kevmoo** [comment]
> Opps - I shouldn't have hit the 'approved' button -

github needs a red flag when you've added comments that automerge is on!

**Pull request merged**